contracts/oracle/PCVOracle.sol

```diff
@@ -189,7 +194,7 @@ contract PCVOracle is IPCVOracle, CoreRefV2 {
         address venue,
         bool isLiquid,
         address newOracle
-    ) external onlyGovernor {
+    ) external onlyGovernor globalLock(1) {
```

Comment:
why bother ? it's onlyGovernor

Reply:
it won't work otherwise because it calls accrue on the underlying deposits which require the system to be at lock level 1, and then lock to level 2